.config/webpack/utils.ts

```diff
@@ -1,42 +1,59 @@
 import fs from 'fs';
+import process from 'process';
+import os from 'os';
 import path from 'path';
 import util from 'util';
-import glob from 'glob';
+import { glob } from 'glob';
 import { SOURCE_DIR } from './constants';
 
-const globAsync = util.promisify(glob);
+
+export function isWSL() {
+  if (process.platform !== 'linux') {
+    return false;
+  }
+
+  if (os.release().toLowerCase().includes('microsoft')) {
+    return true;
+  }
+
+  try {
+    return fs.readFileSync('/proc/version', 'utf8').toLowerCase().includes('microsoft');
+  } catch {
+    return false;
+  }
+} 
 
 export function getPackageJson() {
   return require(path.resolve(process.cwd(), 'package.json'));
 }
 
-export function getPluginId() {
-  const { id } = require(path.resolve(process.cwd(), `${SOURCE_DIR}/plugin.json`));
-
-  return id;
+export function getPluginJson() {
```

Comment:
Like this, this makes much more sense given the companion function `getPackageJson` above.